code/modules/ftl/weaponry/ship_attacks.dm

```diff
@@ -284,6 +284,22 @@
 		smoke.start()
 		qdel(R)
 
+/datum/ship_attack/pirate_boarding_pod
+	cname = "Boarding Pod"
+	projectile_effect = "leaper"
+	hull_damage = 0
+	var/list/boarding_mobs = list(/mob/living/simple_animal/hostile/pirate/space/ranged, /mob/living/simple_animal/hostile/pirate/space)
+	var/amount = 1
+
+/datum/ship_attack/pirate_boarding_pod/damage_effects(turf/epicenter)
```

Comment:
Use carrier_weapon as a parent instead, this is better use of OOP

Reply:
I'm going to need you to explain what you want me to do here.

Reply:
make a child of datum/ship_attack/carrier_weapon because it already has this behavior